.github/workflows/ci-e2e-badger.yaml

```
@@ -41,6 +43,12 @@ jobs:
             ;;
         esac
 
+    - uses: ./.github/actions/verify-metrics-snapshot
+      if: matrix.version == 'v2'
+      with:
+        snapshot: metrics_snapshot_badger
+        artifact_key: badger-metrics-${{ matrix.version }}.txt
```

can we also use the same naming scheme here?
```suggestion
        artifact_key: metrics_snapshot_badger-${{ matrix.version }}
```

The .txt is meaningless here, since the artifact is uploaded as a gzip archive